media-autobuild_suite.bat

```diff
@@ -74,17 +74,17 @@ set ffmpeg_options_builtin=#amf #bzlib #cuda #cuvid #d3d11va #dxva2 ^
 set ffmpeg_options_basic=gmp libmp3lame libopus libvorbis libvpx libx264 libx265
 
 :: options used in zeranoe builds and not present above
-set ffmpeg_options_zeranoe=fontconfig gnutls libass libbluray libfreetype ^
+set ffmpeg_options_zeranoe=fontconfig gnutls libass libaom libbluray libfreetype ^
 libmfx libmysofa libopencore-amrnb libopencore-amrwb libopenjpeg libsnappy ^
-libsoxr libspeex libtheora libtwolame libvidstab libvo-amrwbenc libwavpack ^
+libsoxr libspeex libtheora libtwolame libvidstab libxvid libvo-amrwbenc libwavpack ^
 libwebp libxml2 libzimg libshine gpl openssl libtls avisynth mbedtls
```

Comment:
Add later options to the end. Results in better diffs and doesn't ruin the text limits.